docs/source/daml/reference/packages.rst

```diff
@@ -20,7 +20,7 @@ You can specify a different path for the Daml archive by using the ``-o`` flag:
 
   daml build -o foo.dar
 
-For details on how to upload a Daml archive to the ledger, see the :ref:`deploy documentation <deploy-ref_overview>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
+The rest of this page will focus on how to import a Daml package in other Daml projects.
```

Comment:
why did you delete that? The reference seems very useful.

Reply:
Adding back